apps/client/src/common/hooks/useEventAction.ts

```diff
@@ -253,9 +253,13 @@ export const useEventAction = () => {
           newEvent.timeStrategy = TimeStrategy.LockEnd;
```

Comment:
right now when you submit an empty end time it results in it being set to 00:00:00
I think it should either move the lock to duration or not change the end valure

Reply:
good catch